### DISCUSSION TITLE
Can I run an R inside WSL using Positron?

### DISCUSSION THREAD

**Discussion by filipemsc** [posit-dev/positron#5256] [Q&A] [answered]
Currently, I use Windows 11, but I wrote a code that only works in Linux (don't ask me why, idk either), so I have to use WSL to run it. Is it possible to use WSL to run R in Positron console? Any thoughts on this?

**Comment by testlabauto** [comment, marked as answer]
Hello, thank you for using Positron!  Unfortunately, this is not currently supported but we do have a ticket for adding support in a future release: https://github.com/posit-dev/positron/issues/4366

**Reply by kv9898** [reply]
> Hello, thank you for using Positron! Unfortunately, this is not currently supported but we do have a ticket for adding support in a future release: #4366 

I adapted the `open-remote-wsl` extension for positron in a [fork](https://github.com/kv9898/open-remote-wsl) and published it on Open VSX Registry so it can be downloaded: https://open-vsx.org/extension/kv9898/open-remote-wsl

It works well on my end.

![Open Remote WSL](https://github.com/user-attachments/assets/30dbc136-3ca2-451f-9a3d-58d7c68b5fe6)
